api/src/test/java/org/openmrs/aop/AuthorizationAdviceTest.java

```diff
@@ -63,8 +63,8 @@ public void before_shouldNotifyListenersAboutCheckedPrivileges() {
 		
 		Context.getConceptService().saveConcept(concept);
 		
-		assertThat("listener1", listener1.hasPrivileges, containsInAnyOrder("Manage Concepts", "Get Observations", "Get Concept Attribute Types"));
-		assertThat("listener2", listener2.hasPrivileges, containsInAnyOrder("Manage Concepts", "Get Observations", "Get Concept Attribute Types"));
+		assertThat("listener1", listener1.hasPrivileges, containsInAnyOrder("Manage Concepts", "Get Observations", "Get Concept Attribute Types", "Get Concept Map Types"));
```

Comment:
@mavrk why did it fail? Was it because of your change in ConceptServiceImpl?

The issue you are working on was about adding tests but you also changed the implementation. Can you please remove your changes from the ConceptServiceImpl and this test and just work on testing the ConceptServiceImpl (on which you are doing a great job!). You can refactor code in another issue. Once we have good test coverage showing how it currently works, its safe to change the implementation.